_docs/_latest/use/server.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Server
+order: 007
+---
+
+# {{ page.title }}
+
+Forseti Server is tool to retrieve and update the server settings.
+
+## Running Forseti Server
+
+#### Getting the current server configuration
```

Comment:
Your heading levels are off this should be ### not ####

Reply:
Updated